nifi-docs/src/main/asciidoc/expression-language-guide.adoc

```diff
@@ -884,7 +916,7 @@ Expressions will provide the following results:
 
 
 
-.replaceAll Examples
+.ReplaceAll Examples
```

Comment:
Shouldn't this remain lowercase? Also is it necessary for the markdown table or should it be removed to be consistent with other examples sections?